### PR TITLE
SyndieChemiCompiler_Bugfix

### PR DESCRIPTION
**Pull request by Nitre88**
[bugfix]
## About the PR 
Fixes an issue with the syndicate ChemiCompiler where it could cause runtimes and could not use the pill, drop, or vial functions

## Why's this needed? 
It's currently not able to function as a ChemiCompiler without these important functions. Also ruined my traitor round :(

### CODE REVIEW

**Comment by Nitre88** [comment]
did a bit of last minute testing drip isn't working fixing

File: code/modules/chemistry/chemicompiler_core.dm [hunk 1]
```diff
@@ -393,7 +394,7 @@
 	if(source < minReservoir || source > maxReservoir)
 		throwError(CC_ERROR_INVALID_SX) // Invalid source id.
 		return
-	if(target < minReservoir || target > maxReservoir + 3)
+	if(target < minReservoir || target > maxReservoir && !target == utilReservoir)
```
Comment (Tarmunora) [inline code/modules/chemistry/chemicompiler_core.dm:397]: `utilReservoir` is being defined as a list though - I don't see how this code would actually work?

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/chemistry/chemicompiler_core.dm (hunk 1 above)
Comment (Nitre88) [reply code/modules/chemistry/chemicompiler_core.dm]: thank you and fixed

File: code/modules/chemistry/chemicompiler_core.dm [hunk 2]
```diff
@@ -51,6 +51,7 @@
 	var/heading = "don't touch!!!"
 	var/minStored = 1
 	var/maxStored = 6
+	var/list/utilReservoir = list(11, 12, 13,)
```
Comment (ZeWaka) [inline code/modules/chemistry/chemicompiler_core.dm:54]:
```suggestion
	var/list/utilReservoir = list(11, 12, 13)
```

**Pull request merged**